test_scripts/RC/InteriorVehicleData_cache/003_GetInteriorVD_without_subscribe_2_apps.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0178-GetInteriorVehicleData.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case
+-- 1. Mobile app1 requested GetInteriorVD(module_1, without subscribe parameter) and HMI sent GetInteriorVD to HMI
+-- 2. Mobile app2 GetInteriorVD(module_1, without subscribe parameter) request
+-- SDL must
+-- 1. send GetInteriorVD(module_1, without subscribe parameter) request to HMI
```

Comment:
SDL send two, one for app1, one for app2

Reply:
@yang1070 , sending of the first request is described in point 1. Mobile app1 requested GetInteriorVD(module_1, without subscribe parameter) **and HMI sent GetInteriorVD to HMI** but need update to  **and SDL sents GetInteriorVD to HMI**

Reply:
@yang1070 , updated according to my comment above.